pkg/capabilities/crypto/crypto.go

```diff
@@ -0,0 +1,68 @@
+package crypto
+
+import (
+	"fmt"
+
+	cap "github.com/kubewarden/policy-sdk-go/pkg/capabilities"
+
+	"github.com/mailru/easyjson"
+	jwriter "github.com/mailru/easyjson/jwriter"
+)
+
+type CryptoHost struct {
+	cap.Host
+}
```

Comment:
Considering your latest changes, I think this type can be removed.
```suggestion
}
```